app/controllers/calendars_controller.rb

```diff
@@ -15,10 +15,10 @@ def create
   private
 
   def plan_params
-    params.require(:calendars).permit(:date, :plan)
+    params.require(:Calendars).permit(:date, :plan)
```

Comment:
```suggestion
    params.require(:calendars).permit(:date, :plan)
```
こちらの`Calendars`に関しては頭文字が小文字で構いません。
小文字に戻しておきましょう。